tests/test_cut_diff.py

```diff
@@ -1071,6 +1118,48 @@ def test_report(self):
                 "\n\nLinks: \n\nThe changes in This is a Test are as follows:\n\n0 New Shots\n\n\n0 Omitted Shots\n\n\n0 Reinstated Shot\n\n\n0 Cut Changes\n\n\n0 Rescan Needed\n\n\n"
             )
         )
+        # Test csv report
+        _, csv_path = tempfile.mkstemp(suffix=".csv")
+        track_diff.write_csv_report(csv_path, "This is a Test", [mock_cut_url])
+        with open(csv_path, newline="") as csvfile:
```

Comment:
I had to google it, so maybe it's worth a comment that csv does its own new line handling and that's why we use `newline=""`